composer.json

```diff
@@ -27,6 +27,7 @@
 		"christophwurst/kitinerary-sys": "^0.1.0",
 		"ezyang/htmlpurifier": "4.14.0",
 		"gravatarphp/gravatar": "^2.0",
+		"jeremykendall/php-domain-parser": "^5.7",
```

Comment:
this is the newest version compatible. it has a different API. Tests are failing.